NEWS.md

```diff
@@ -4,6 +4,19 @@
 
 ## data.table [v1.17.99](https://github.com/Rdatatable/data.table/milestone/35)  (in development)
 
+### BREAKING CHANGE
+
+1. Rolling functions `frollmean` and `frollsum` used to treat `Inf` and `-Inf` as `NA` when using default `algo="fast"`. It has been changed now and infinite values are not treated as `NA` anymore. If your input into those functions has `Inf` or `-Inf` then you will be affected by this change. As a result, argument that controls behavior of `NA`s has been renamed from `hasNA` to `has.nf` (_has non-finite_), as of 1.18.0 both will work, former with a warning.
```

Comment:
I prefer to lead with what's changed, then describe the old behavior, should make it easier to read.
```suggestion
1. Rolling functions `frollmean` and `frollsum` distinguish `Inf`/`-Inf` from `NA` when `algo="fast"` (previously then were considered the same). If your input into those functions has `Inf` or `-Inf` then you will be affected by this change. As a result, the argument that controls the handling of `NA`s has been renamed from `hasNA` to `has.nf` (_has non-finite_). `hasNA` continues to work with a warning, for now.
```

Reply:
it would also be good to give a minimal example of before/after output directly here in the NEWS.